frontend/app/src/pages/people/Person.tsx

```diff
@@ -190,7 +190,7 @@ export default function Person(props: PersonProps) {
           </div>
           <div style={{ padding: 16 }}>
             <DTitle>{owner_alias}</DTitle>
-            {description && (
+            {description && description !== 'description' &&(
               <DDescription>{description === 'description' ? '' : description}</DDescription>
```

Comment:
looks like this already has a check on the line below, does that check not work?

Reply:
Apologies @kevkevinpal, I realized I committed to the wrong fork. Give me 10 minutes, and I'll have the correct version pushed. Thanks for pointing it out!